site/content/en/docs/components/core/hook.md

```diff
@@ -1,8 +1,41 @@
 ---
 title: "Hook"
-weight: 30
-description: >
-  
+weight: 60
+description: PROW (Prow Hook) is a webhook service for Kubernetes that listens for events from Github, Gitlab, and other sources, and then triggers actions in response to those events. It is commonly used to automate tasks such as building and deploying code, running tests, and managing pull requests.
 ---
+## Usage / How to enable 
 
-This is a placeholder page. Some contents needs to be filled.
+To enable PROW Hook for Github and Gerrit, you will need to configure the webhooks in each platform and configure the Prow Hook service to listen for events from those webhooks.
+
+Here are the general steps you would need to take:
+
+1. Configure webhooks in Github:
+ -  In your Github repository, navigate to the "Settings" tab and select "Webhooks" from the left sidebar.
+ -  Click the "Add webhook" button.
+ -  In the "Payload URL" field, enter the URL of your Prow Hook service. This URL should be publicly accessible.
+ -  Select "application/json" for the "Content type" field.
+ - Select the events you want to trigger the webhook (e.g. push, pull request, etc).
+ - Click the "Add webhook" button to save the configuration.
+
+2. Configure webhooks in Gerrit:
+ - In your Gerrit repository, navigate to the "Settings" tab and select "Webhooks" from the left sidebar.
+ - Click the "Add webhook" button.
+ - In the "URL" field, enter the URL of your Prow Hook service. This URL should be publicly accessible.
+ - Select the events you want to trigger the webhook (e.g. change merged, change abandoned, etc).
+ - Click the "Add webhook" button to save the configuration
+ 
+3. Configure Prow Hook service to listen for events from Github and Gerrit:
+ - In your Prow Hook configuration file, you need to add the following fields to enable the webhooks of Github and Gerrit.
+```
+hooks:
+- name: gerrit
+  type: gerrit
+  endpoint: https://your-gerrit-instance.com/a/changes/
+- name: github
+  type: github
+  endpoint: https://api.github.com
+``` 
```

Comment:
I don't think this configuration exists

Reply:
removed it , I haven't got chance to test this .